### DISCUSSION TITLE
Accuracy Indicator

### DISCUSSION THREAD

**Discussion by xeruf** [komoot/photon#700] [Q&A]
As requested in this OSM Help thread as well, an indication of accuracy when geocoding could be helpful:
https://help.openstreetmap.org/questions/12057/bulk-geo-coding-of-business-addresses

Somewhat related to https://github.com/komoot/photon/issues/626

**Comment by lonvia** [comment]
Photon returns a `type` property that tells you if the result is an address, a street, a city etc. The possible values are the same as the [layer filter values](https://github.com/komoot/photon#filter-results-by-layer). There is no distance accuracy.
